website/docs/docs/use-dbt-semantic-layer/setup-sl.md

```diff
@@ -0,0 +1,106 @@
+6. After you’ve entered your credentials, you should see connection information that will allow you to connect to downstream tools. If the tool you are using can connect with JDBC, you can save the **JDBC URL** or each of the individual components provided (e.g., environment id, host). Alternatively, if the tool you connect to uses the Semantic Layer GraphQL API, save the GraphQL API host information.
+7. Next, go back to the **Project Details** page and select **Generate Service Token** to create a Semantic Layer service token. Save this token for later.
+8. You’re done 🎉! The semantic layer should is now enabled for your project. 
+-->
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+With the dbt Semantic Layer, you'll be able to centrally define business metrics, reduce code duplication and inconsistency, create self-service in downstream tools, and more. Configure the dbt Semantic Layer in dbt Cloud to connect with your integrated partner tool. 
+
+## Prerequisites
+
+Before you set up the dbt Semantic Layer, make sure you meet the following:
+
+<Snippet path="sl-prerequisites" />
+
+## Set up dbt Semantic Layer
+
+:::tip
+If you're using the legacy Semantic Layer, we **highly** recommend you [upgrade your dbt version](/docs/dbt-versions/upgrade-core-in-cloud) to dbt v1.6 or higher to use the new dbt Semantic Layer. Refer to the dedicated [migration guide](/guides/migration/sl-migration) for more info.
+
+:::
+
+Before you continue with the following steps, you **must** have a multi-tenant dbt Cloud account hosted in North America. 
+ * Team and Enterprise accounts can set up the Semantic Layer and [Discovery API](/docs/dbt-cloud-apis/discovery-api) in the integrated partner tool to import metric definitions. 
+ * Developer accounts can query the Proxy Server using SQL but won't be able to browse dbt metrics in external tools, which requires access to the Discovery API.
+
+You can set up the dbt Semantic Layer in dbt Cloud at the environment level by following these steps:
+
+1. Login to your dbt Cloud account
+2. Go to **Account Settings**, and then **Service Tokens** to create a new [service account API token](/docs/dbt-cloud-apis/service-tokens). Save your token somewhere safe. 
+3. Assign permissions to service account tokens depending on the integration tool you choose. You can review the [integration partner documentation](https://www.getdbt.com/product/semantic-layer-integrations) to determine the permission sets you need to assign.
+4. Go to **Deploy** and then **Environment**, and select your **Deployment** environment.
+5. Click on **Settings** on the top right side of the page.
+6. Click **Edit** on the top right side of the page.
+7. Select dbt version 1.2 or higher.
+8. Toggle the Semantic Layer **On**.
```

Comment:
the bold suggests that there's a UI label with "On". is that true? 

if there's an option name, suggest using that instead, like "Enable the **<blank>** option." 

```suggestion
8. Toggle the Semantic Layer **On**.
```

Reply:
yes it's labeled with On!